forms/models.py

```diff
@@ -62,20 +73,68 @@ class Meta(AbstractFormField.Meta):
     )
 
     panels = AbstractFormField.panels + [
+        FieldPanel('placeholder'),
         FieldPanel('append_to_subject'),
         FieldPanel('use_as_reply_to'),
     ]
 
 
+class FieldGroup(ClusterableModel, Orderable):
```

Comment:
In spite of this being an `Orderable`, I'm having a hard time reordering these, locally. Or at least getting the order to stick? If I change the order in the admin, then publish, the order on the rendered page doesn't seem to change (and reverts back to its previous state in the admin). Are you seeing this? I'm not sure what's going on, actually. Might be something weird on my end?

Reply:
@chigby 

I think I've seen this before - I've added an explicit `sort_order` to the model's Meta ordering, and this looks to have solved the issue.